R/utils.R

```diff
@@ -103,6 +103,11 @@ assert_raw <- function(x, name = deparse(substitute(x))) {
   }
 }
 
+assert_function <- function(x, name = deparse(substitute(x))) {
```

Comment:
This already exists on line 53 😁

Reply:
Removed the duplicate: 87b86e994f946e71846ca91ee3a797c289a3ec6b